Marlin/MarlinSerial.cpp

```diff
@@ -310,3 +315,155 @@ MarlinSerial customizedSerial;
 #if defined(USBCON) && ENABLED(BLUETOOTH)
   HardwareSerial bluetoothSerial;
 #endif
+
+#if ENABLED(EMERGENCY_PARSER)
+
+  #define EMERGENCY_CODES "M112,M108,M410"
```

Comment:
You are right. The idea was to define it here where we exactly know what commands we are parsing.
But instead of using an other indirection in Configuration_adv.h i will put the info directly into language.h. Making this mini parser configurable would be a dream, but it isn't. A definition in Configuration_adv.h could make the impression it would be configurable - i want to avoid that.